modules/nw-ovn-kubernetes-list-database-contents.adoc

```diff
@@ -33,201 +30,184 @@ $ oc get po -n openshift-ovn-kubernetes
+                                                            tcp        172.30.148.52:444       10.0.0.4:9203,10.0.0.5:9203
+                                                            tcp        172.30.148.52:445       10.0.0.4:9204,10.0.0.5:9204
+                                                            tcp        172.30.148.52:446       10.0.0.4:9205,10.0.0.5:9205
+2a33a6d7-af1b-4892-87cc-326a380b809b    Service_openshif    tcp        172.30.67.219:9091      10.129.2.16:9091,10.131.0.16:9091
+                                                            tcp        172.30.67.219:9092      10.129.2.16:9092,10.131.0.16:9092
+                                                            tcp        172.30.67.219:9093      10.129.2.16:9093,10.131.0.16:9093
+                                                            tcp        172.30.67.219:9094      10.129.2.16:9094,10.131.0.16:9094
+f56f59d7-231a-4974-99b3-792e2741ec8d    Service_openshif    tcp        172.30.89.212:443       10.128.0.41:8443,10.129.0.68:8443,10.130.0.44:8443
+08c2c6d7-d217-4b96-b5d8-c80c4e258116    Service_openshif    tcp        172.30.102.137:2379     10.0.0.3:2379,169.254.169.2:2379,10.0.0.5:2379
+                                                            tcp        172.30.102.137:9979     10.0.0.3:9979,169.254.169.2:9979,10.0.0.5:9979
+60a69c56-fc6a-4de6-bd88-3f2af5ba5665    Service_openshif    tcp        172.30.10.193:443       10.129.0.25:8443
+ab1ef694-0826-4671-a22c-565fc2d282ec    Service_openshif    tcp        172.30.196.123:443      10.128.0.33:8443,10.129.0.64:8443,10.130.0.37:8443
+b1fb34d3-0944-4770-9ee3-2683e7a630e2    Service_openshif    tcp        172.30.158.93:8443      10.129.0.13:8443
+95811c11-56e2-4877-be1e-c78ccb3a82a9    Service_openshif    tcp        172.30.46.85:9001       10.130.0.16:9001
+4baba1d1-b873-4535-884c-3f6fc07a50fd    Service_openshif    tcp        172.30.28.87:443        10.129.0.26:8443
+6c2e1c90-f0ca-484e-8a8e-40e71442110a    Service_openshif    udp        172.30.0.10:53          10.128.0.13:5353,10.128.2.6:5353,10.129.0.39:5353,10.129.2.6:5353,10.130.0.11:5353,10.131.0.9:5353
+
 ----
 +
 [NOTE]
 ====
 From this truncated output you can see there are many OVN-Kubernetes load balancers. Load balancers in OVN-Kubernetes are representations of services.
 ====
 
+. Run the following command to display the options available with the command `ovn-nbctl`:
++
+[source,terminal]
+----
+$ oc exec -n openshift-ovn-kubernetes -it ovnkube-node-55xs2 \
+-c northd ovn-nbctl --help
+----
+
+
+//
+//[source,terminal]
+//----
+//$ oc get po -o wide -n openshift-ovn-kubernetes | grep  10.0.0.5 | grep -v ovnkube-node
+//----
+//+
+//.Example output
+//[source,terminal]
+//----
+//ovnkube-control-plane-8444dff7f9-k64b7   2/2     Running   2 (28m ago)   44m   10.0.0.5     ci-ln-t487nnb-72292-mdcnq-master-0         <none>           <none>
+//----
+//+
+//The `ovnkube-master-gt4ms` pod runs on IP Address 10.0.242.240.
```

Comment:
what's mean this line? guess it can be removed..

Reply:
They should have been deleted.